go-controller/pkg/ovn/ovn.go

```diff
@@ -611,3 +649,22 @@ func macAddressChanged(oldNode, node *kapi.Node) bool {
 	macAddress := node.Annotations[OvnNodeManagementPortMacAddress]
 	return oldMacAddress != macAddress
 }
+
+// noHostSubnet() compares the no-hostsubenet-nodes flag with node labels to see if the node is manageing its
+// own network.
+func (oc *Controller) noHostSubnet(node *kapi.Node) (bool, error) {
+
+	if config.Kubernetes.NoHostSubnetNodes == "" {
+		return false, nil
+	}
+
+	nodeLabelSelector, err := metav1.ParseToLabelSelector(config.Kubernetes.NoHostSubnetNodes)
```

Comment:
@JacobTanenbaum Let's put this bit into config.go so we only do it once and hard-error on startup if the label selector is malformed. See the comments on config.go

Reply:
fixed